types/src/Mooclet/interfaces.ts

```diff
@@ -0,0 +1,152 @@
+/**********************************************************************
+ * types for the Mooclet services. I don't know where to put these so they're here!
+ */
+export interface MoocletProxyRequestParams {
```

Comment:
Is this type going to be used on the frontend also? Only shared types should go here.